packages/components/src/VirtualizedList/RowLarge/RowLarge.component.js

```diff
@@ -44,7 +44,7 @@ function LargeInnerRowLoading({ columns, rows }) {
 					<div key={index} className={theme['loading-inner-column']}>
 						{Array(rows)
 							.fill(0)
-							.map((_, innerIndex) => (
+							.map((__, innerIndex) => (
```

Comment:
`_` is already declared before

Reply:
and "row" instead?

Reply:
It's not used, it I set a name I'll have a new lint error, because the var is declared but not used